number_theory/modular.h

```diff
@@ -46,6 +48,27 @@ struct ModulusWrapper {
 
 }  // namespace modular_internal
 
+// Returns the modular inverse of |number| in modulo |mod| if exists.
+// Otherwise, throws an invalid_argument exception.
+template <typename T>
+T inverse_mod(const T &number, const T &mod) {
+  static_assert(std::numeric_limits<T>::is_integer);
+  using Signed_T = std::make_signed_t<T>;
+  T n = modular_internal::normalize(number, mod);
```

Comment:
Do we really need to call normalize before exgcd?

Reply:
I thought it would result in a better performance in some cases where the number is too large compared to mod, but I guess I was wrong. exgcd already probably runs in O(log min(number, mod)) time.